src/test/resources/unrc/dose/Goldenberg.feature

```diff
@@ -0,0 +1,102 @@
+#Authors: Borda, Celi, de Prada, Goldenberg.
+
+Scenario: Admin user wants to create a new valid test based challenge
+  Given the user “root” is already logged on
+  And the user has admin permission
+  And the text of the challenge is
+  “
+    public class Module{
+      public static int module(int a){
+        if(a >= 0){
+            return -a;
+        }
+        else{
+          return a;
+        }
+      }
+    }
+  ”
```

Comment:
I think that we should use triple tilde when using multi line strings in cucumber syntax ()